src/status_im/i18n/i18n.cljs

```diff
@@ -64,3 +65,10 @@
   (.-locale i18n))
 
 (def format-currency goog.i18n/format-currency)
+
+(defn format-members [count]
```

Comment:
how its related to i18n ?

Reply:
This one is not at all related. I place this in the right ns